public/docs/_examples/dependency-injection/dart/web/heroes/mock-heroes.dart

```diff
@@ -0,0 +1,15 @@
+// #docregion
+import "hero.dart";
+
+List<Hero> HEROES = [
+  new Hero()..id = 11..isSecret = false..name = "Mr. Nice",
+  new Hero()..id = 12..isSecret = false..name = "Narco",
```

Comment:
Is this how darformat formats this? I got the impression this was changed to one line for each cascade. Could be wrong though.